src/BlockTracer.h

```diff
@@ -39,32 +39,58 @@ class cBlockTracer abstract
 
 		/** Called on each block encountered along the path, including the first block (path start)
 		When this callback returns true, the tracing is aborted.
+		Prefer Vector3i overload
```

Comment:
Ideally, this would be removed entirely and a manual binding added to return a deprecated message to plugins and do the necessary conversion. If the old form is used anywhere in the C++ code, it needs to be converted to the new form as part of this refactor.

Reply:
Does manual binding work next to automatic ones?

Reply:
I think @madmaxoft or @peterbell10 know more about the bindings than me, maybe they could describe the process for choosing manual/automagic bindings better.

Reply:
Should a manual binding with both deprecated and not deprecated bindings be put in ManualBindings.cpp or in DeprecatedBindings.cpp ?

Reply:
I don't think this file is automatically exported to lua so there are only manual bindings.  It should go in `ManualBindings.cpp`.

That said, I would be happy enough merging a PR where the lua API doesn't use `Vector3` yet. It just wouldn't be enough to close #4668.